packages/ui/src/kit/VAccordion/VAccordionItem/VAccordionItem.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    :tabindex="0"
+    :class="itemClassName"
+    @click="onClick"
+  >
+    <div class="accordion-header">
+      <div class="accordion-title">
+        <slot name="title" />
+      </div>
+      <icon-button-atom
+        :icon="icon"
+        width="20"
+        class="accordion-item-icon"
+      />
+    </div>
+    <div
+      v-show="isOpened"
+      class="accordion-content"
+    >
+      <slot />
+    </div>
+  </div>
+</template>
+
+<script>
+import IconButtonAtom from '@/atom/icon-button-atom/icon-button-atom.vue';
+import ThemeMixin from '@/mixins/ThemeMixin';
+
+export default {
+  name: 'VAccordionItem',
+  props: {
+    className: {
+      type: String,
+      default: null,
+    },
+  },
+  data: () => ({
+    isOpened: false,
+  }),
+  computed: {
+    icon() {
+      return this.isOpened ? 'chevron-up' : 'chevron-down';
+    },
+    itemClassName() {
+      return `v-accordion-item ${this.className || ''}`;
```

Comment:
the [same](https://github.com/endpass/endpass-core/pull/302/files#diff-af7a7795a5fa6ab723d97d7714af5312R20)